frontend/src/views/pages/inventory/InventoryAdd.vue

```diff
@@ -0,0 +1,226 @@
+<template>
+  <div>
+    <CCard v-if="dealership">
+      <CCardBody>
+        <h1>Adding New Vehicle</h1>
+        <h5>Dealership: {{ dealership.name }}</h5>
+        <p>In this page you can add a new vehicle to the inventory of the specified dealership. 
+          Vehicle model is defined in the dealership's parameters 
+          <router-link v-if="userHasPermissions('View Dealerships')" :to="`/dealerships/${dealership._id}`">here</router-link> 
+          under "Vehicle Properties"</p>
+      </CCardBody>
+    </CCard>
+    <CRow>
+      <CCol>
+        <CCard v-if="vehicleProperties">
+          <CCardBody>
+            <h3>Vehicle Properties</h3>
+            <hr>
+            <CForm ref="vehicleForm" @submit.prevent="submitForm">
+              <div v-for="property in vehicleProperties" :key="property._id">
+                <CInput 
+                  v-if="property.inputType == 'Text'"
+                  :label="property.label"
+                  :name="property.key"
+                  :required="property.isRequired"
+                />
+                <CInput 
+                  v-if="property.inputType == 'List'"
+                  :label="property.label"
+                  :name="property.key"
+                  description="Separate list item by semi-colons ';'"
+                  :required="property.isRequired"
+                />
+                <CSelect
+                  v-if="property.inputType == 'Options'"
+                  :label="property.label"
+                  :options="property.options"
+                  :name="property.key"
+                  :required="property.isRequired"
+                />
+                <CInput 
+                  v-if="property.inputType == 'Date'"
+                  type="date"
+                  :label="property.label"
+                  :name="property.key"
+                  :required="property.isRequired"
+                />
+                <!-- <CFormGroup
+                  wrapperClasses="input-group pt-2 pb-3"
+                  v-if="property.inputType == 'Date'"
+                >
+                  <template #prepend-content>
+                    <CIcon name="cil-calendar"/>
+                  </template>
+                  <template #label>
+                    {{ property.label }}
+                  </template>
+                  <template #input>
+                    <masked-input
+                      type="text"
+                      :name="property.key"
+                      class="form-control"
+                      :mask="[/0|1|2|3/, /\d/, '-', /0|1|2|3/, /\d/, '-', /\d/, /\d/, /\d/, /\d/]"
+                      :guide="true"
+                      placeholderChar="_"
+                      :showMask="true"
+                      :keepCharPositions="true"
+                      :required="property.isRequired"
+                    />
+                  </template>
+                </CFormGroup> -->
```

Comment:
Remove commented code

Reply:
Will create a pull request next sprint that will clean up all the console.logs and error cases for API requests